packages/react-core/src/components/Form/FormGroup.tsx

```diff
@@ -40,7 +40,7 @@ export interface FormGroupProps extends Omit<React.HTMLProps<HTMLDivElement>, 'l
   /** Icon displayed to the left of the helper text when the field is invalid. */
   helperTextInvalidIcon?: React.ReactNode;
   /** ID of the included field. It has to be the same for proper working. */
-  fieldId: string;
+  fieldId?: string;
```

Comment:
If we decide to have this prop continue working as-is, i.e. setting that `aria-labelledby` on FormGroup, then this description could use updating. Something along the lines of below, but I'd hold off until Nicole responds to my other comment regarding this prop.

"ID of an individual field or a group of multiple fields. Required when a role of "group" or "radiogroup" is passed in. If only one field is included, its ID attribute and this prop must be the same."